client/core/trade.go

```diff
@@ -36,12 +36,13 @@ func (err ExpirationErr) Error() string { return string(err) }
 // A matchTracker is used to negotiate a match.
 type matchTracker struct {
 	db.MetaMatch
-	failErr     error
-	refundErr   error
-	prefix      *order.Prefix
-	trade       *order.Trade
-	counterSwap asset.AuditInfo
-	id          order.MatchID
+	failErr        error
+	refundErr      error
+	prefix         *order.Prefix
+	trade          *order.Trade
+	counterSwap    asset.AuditInfo
+	contractExpiry time.Time // for one time logging in isRefundable, may never be set
```

Comment:
Perhaps a little hackish since the value stored isn't important except for debugging and as a flag, but it works to limit the "not yet expired" log message to at most one time per match.